src/object/trigger_manager.c

```diff
@@ -4225,6 +4232,24 @@ tr_find_trigger (const char *name)
 
   if (trigger_table_find (name, &object) == NO_ERROR)
     {
+      /* This is the case when the loaddb utility is executed with the --no-user-specified-name option as the dba user. */
+      if (db_get_client_type () == DB_CLIENT_TYPE_ADMIN_UTILITY && prm_get_bool_value (PRM_ID_NO_USER_SPECIFIED_NAME))
+	{
+	  if (object == NULL)
```

Comment:
How likely is object to be null in trigger_table_find()?
It would be better to change the order of checking if() statements.
```
      if (object == NULL)
	{
	  if (db_get_client_type () == DB_CLIENT_TYPE_ADMIN_UTILITY && prm_get_bool_value (PRM_ID_NO_USER_SPECIFIED_NAME))
```

Reply:
Okay, I'll change it. It seems good to check for NULL first.